Lib/test/test_free_threading/test_memoryview.py

```diff
@@ -0,0 +1,36 @@
+import multiprocessing.shared_memory
+from threading import Thread
+import unittest
+
+from test.support import threading_helper
+
+
+class TestBase(unittest.TestCase):
+    pass
+
+
+def do_race():
+    """Repeatly access the memoryview for racing."""
+    n = 100
+
+    obj = multiprocessing.shared_memory.ShareableList("Uq..SeDAmB+EBrkLl.SG.Z+Z.ZdsV..wT+zLxKwdN\b")
+    threads = []
+    for _ in range(n):
+        threads.append(Thread(target=obj.count, args=(1,)))
+
+    for t in threads:
+        t.start()
+
+    for t in threads:
+        t.join()
```

Comment:
We have a utility function for this:
```suggestion
    with threading_helper.start_threads(threads):
        pass
```

Reply:
Hi @ZeroIntensity ,

Thanks very much for your warm suggestion! 😊

I have replaced this part with `threading_helper.start_threads`.

Best Regards,
Edward